basic.md

```diff
@@ -702,6 +702,17 @@ Currently not implemented, does nothing
 
 Currently not implemented, does nothing
 
+### `END`
+
+End execution of the program silently. Note that `SYSTEM` seems to exhibit the same behaviour.
```

Comment:
`SYSTEM` is an alias to the `END` reserved keyword, so it should either be in the same list as `END` in alphabetical order, or this description should say that `SYSTEM` is an alias to `END`

Reply:
I didn't know if it was a strict alias or just operated the same. I've updated the text to state that it is an alias.